src/ContentRepository/ApplicationModel/ActionFramework.cs

```diff
@@ -168,17 +168,25 @@ public static IEnumerable<ActionBase> GetActions(Content context, string scenari
             return GetActions(context, scenario, scenarioParameters, backUrl);
         }
 
-        public static IEnumerable<ActionBase> GetActions(Content context, string scenario, string scenarioParameters, string backUri)
+        public static IEnumerable<ActionBase> GetActions(Content context, string scenarios, string scenarioParameters, string backUri)

```

Comment:
We should keep the original API to avoid break (this is public api, upper layers may use it). A new method override would be better that accepts a **string array for scenarios**. We can simply leave out the _scenarioparameters_ parameter there. If you have that, use the original api for a single scenario.

It should not be the responsibility of this method to split scenarios.